packages/dd-trace/src/profiling/profilers/events.js

```diff
@@ -240,10 +236,10 @@ class EventsProfiler {
 
     return new Profile({
       sampleType: [timeValueType],
-      timeNanos: dateOffset + BigInt(Math.round(durationFrom * MS_TO_NS)),
+      timeNanos: endDate.getTime() * MS_TO_NS,
```

Comment:
Yes, this was intentionally changed from emitting the start of the period to emitting the end of the period; latter is correct.